go.sum

```diff
@@ -1,6 +1,7 @@
 github.com/Microsoft/go-winio v0.4.12/go.mod h1:VhR8bwka0BXejwEJY73c50VrPtXAaKcyvVC4A4RozmA=
 github.com/aws/amazon-ecs-agent v1.26.0 h1:kT/JZYXpCUXSjNaTWrYjlCPayWCV6dXaMQKOYq5JLfo=
 github.com/aws/amazon-ecs-agent v1.26.0/go.mod h1:as/yyU3OZwCtKXO0Bv9dieYPupNo7jnNNZRNQTu9Row=
+github.com/aws/amazon-ecs-cli v1.13.0 h1:cOOwtpChp2JhDK7PkIoHN5OKNhCPSQhaL7n3Pz7nIYk=
```

Comment:
can this be removed? I don't see anywhere new that's using it

Reply:
huh.... this is autogenerated.. not sure why it didn't get auto-removed when I re-built after removing it from go.mod

Reply:
fixed